Add June 2024 newsletter (#2157)
writethedocs/www: docs/blog/newsletter-june-2024.rst

:og:image: https://www.writethedocs.org/_static/logo-opengraph.png

.. post:: June 05, 2024
  :tags: newsletter

#########################################
Write the Docs Newsletter – June 2024
#########################################

Howdy, documentarians. Maybe it's the ups and downs of the weather (from scorching hot to soaking wet), maybe it's all the reports of AI coming for jobs, but it seems everything is in flux these days. It's important to me to see the way documentarians support each other through these shifts. Maybe the only thing permanent these days is change. That and another monthly newsletter packed full of insights from our `Slack community </slack/>`__.

> If you're interested what's been happening recently in the community, check out the `Q2 Community Board quarterly update </blog/2024-Q2-community-board/>`__. We also saw the first large `gathering of the Write the Docs community in Kenya <https://wtdkenya.hashnode.dev/write-the-docs-kenya-summit-2024>`__.

Reviewer: ```suggestion
If you're interested in what's been happening recently in the community, check out the `Q2 Community Board quarterly update </blog/2024-Q2-community-board/>`__. We also saw the first large `gathering of the Write the Docs community in Kenya <https://wtdkenya.hashnode.dev/write-the-docs-kenya-summit-2024>`__.
```

And coming up in the future, we have the Atlantic 2024 conference. Speakers will be announced any day now, so `follow the website </conf/atlantic/2024/>`__ and be sure to `grab a ticket </conf/atlantic/2024/tickets/>`__ to see the talks.

This month's newsletter offers ideas about how to demonstrate the value of what you do, tips on transitioning from working alone to being on a team, a resurrection of the old debate about using sentence or title case, and tips on optimizing the search on your docs website.

---------------------------
Demonstrate value from docs
---------------------------

> How do documentarians justify their salaries (and promotions or salary increases)? Do individual documentarians perceive the value that their employers gain from their work? The value of documentation and your role in producing good documentation are actually wide ranging. To justify your employment, you may have to quantify some aspects of your role.

Reviewer 2: I *think* there should be a hyphen in "wide ranging"

Author: I'm used to not having hyphens when the phrase isn't before a noun, but it does seem it's more common with the hyphen no matter where in the sentence.
```suggestion
How do documentarians justify their salaries (and promotions or salary increases)? Do individual documentarians perceive the value that their employers gain from their work? The value of documentation and your role in producing good documentation are actually wide-ranging. To justify your employment, you may have to quantify some aspects of your role.
```

+++++++
Support
+++++++

- Assists users with setting up, learning, and troubleshooting a product. Saves support personnel from dealing with unnecessary support calls.
- Contributes to the support team by helping support personnel do their work and eliminating duplicate work. 

One resource for ideas about adding value to your documentation is Kathy Sierra's book `Badass: Making Users Awesome <https://www.oreilly.com/library/view/badass-making-users/9781491919057/>`__, which discusses helping your customers use your product.

+++++++++++++++++++
Sales and marketing
+++++++++++++++++++

- Turns online browsing into actual sales. When evaluating competitive products, potential users can browse and make decisions based on publicly available documentation. 

> - Provides sales resources to help drive sales by answering questions about features and how to use the product. This can lower customer acquisition cost.

Reviewer: ```suggestion
- Helps drive sales by answering questions about features and how to use the product. This can lower customer acquisition cost.
```

Reviewer: Judging by the typical format of these bullet points, perhaps this line is better as

"Helps drive sales and lowers acquisition cost. Answers questions about features and how to use the product."

- Enhances branding and the company’s reputation. Consistent application of company style guides along with voice and tone presents a good face for the company. 
- Retains existing customers and reduces customer turnover.

++++++++++++++++++++++++
Developers and engineers
++++++++++++++++++++++++

> - Improves developer productivity. A `GitHub Octoverse report <https://octoverse.github.com/2021/creating-documentation/>`_ from 2021 claimed good documentation boosts developer productivity by 50%. May be particularly when documenting APIs or SDKs.

Reviewer: ```suggestion
- Improves developer productivity. A `GitHub Octoverse report <https://octoverse.github.com/2021/creating-documentation/>`_ from 2021 claimed good documentation boosts developer productivity by 50%. This may be particularly true when documenting APIs or SDKs.
```

At the same time, you'll have to let go of some independence and share responsibility. You probably won't get to choose the toolchain, and you'll have to follow processes and style guides that reflect more than just your perspective. If the team is comfortable with the processes and tools they're using, it can be difficult to make improvements even if something isn't working well. You may not get every project you want when the work is distributed among several writers. You may be frustrated when quality differs among writers. Also, some writers on docs teams feel more exposed to layoffs since there are others on the team who do the same kind of work.

> When you're the new writer on a docs team after working solo, it's important to be patient and assume good faith. Be kind to those with less seniority, take a servant leader approach when needed, and try to remember that you have plenty to learn. For more tips based on this discussion, read `Things I remind myself when working with others <https://passo.uno/tips-working-tech-writers-team/>`__ by Fabrizio Ferri Benedetti.

Reviewer: I'm not the best at grammar...should it be "servant-leader approach"?

Author: It certainly can be. I've found some versions English are less prone to that kind of thing, but clarity can always be helpful.
```suggestion
When you're the new writer on a docs team after working solo, it's important to be patient and assume good faith. Be kind to those with less seniority, take a servant-leader approach when needed, and try to remember that you have plenty to learn. For more tips based on this discussion, read `Things I remind myself when working with others <https://passo.uno/tips-working-tech-writers-team/>`__ by Fabrizio Ferri Benedetti.
```

One person in a recent discussion emphasized the importance of consistency: "Pick one, stick it in your style guide, then ruthlessly follow whatever you decide." Others echoed this, stressing the need for a documented and standardized approach.

Sentence case is often favored for simplicity and accessibility, especially for non-professional writers, a practical consideration in workplaces where technical writers are scarce. Another person argued against title case, highlighting the complexity: "Title case rules in English vary wildly." Sentence case, being simpler, reduces editorial workload and aids translators.

On the other hand, some agreed on the value of title case for its potential to add clarity and emphasis. If the decisions are being made by professional writers, casing should not be too difficult. There are tools (such as `Capitalize My Title <https://capitalizemytitle.com/>`__) that can help even non-professionals adapt to a standard.

> When choosing between a title and sentence case, it's crucial to consider our audience's needs and the text's context. As one member said, "It's the job of the formatting and layout to clearly distinguish headings from body text." This approach ensures that our design decisions are driven by user requirements.

Reviewer: ```suggestion
When choosing between title and sentence case, it's crucial to consider our audience's needs and the text's context. As one member said, "It's the job of the formatting and layout to clearly distinguish headings from body text." This approach ensures that our design decisions are driven by user requirements.
```

The debate continues, but the main principle remains: choose a style, document it in your style guide, and apply it consistently. This approach enhances clarity and supports a seamless, accessible user experience.

-----------------------------------------------
Search platform tips for documentation websites
-----------------------------------------------

With Algolia/DocSearch holding a top spot among documentation search platforms, writers shared their alternative preferred search platforms in a recent community discussion. Popular alternate choices included Inkeep, Typesense, Meilisearch, Scroll Viewport, lunr.js, and Pagefind.

One writer detailed their approach to optimizing content with AI by asking AI to list how human and non-human readers might misinterpret the content and describe what a user might be doing that led them to the document. Another contributor highlighted the efficiency of Pagefind while acknowledging the complexity of customizing its UI.

> A key, unanimous takeaway was the critical role of well-maintained, clear, and current content in achieving effective search results. The discussion underscored the importance of ongoing content optimization and SEO to ensure users find relevant information quickly and accurately.

Reviewer: I think the 3rd paragraph of this section should be moved to the 2nd paragraph and the 2nd paragraph should be the 3rd paragraph. The content flows more logically that way: intro --> key point about content optimization --> example of using AI for optimizing content. Thoughts?

Author: I liked the flow of "one person said X, another Y ... a unanimous takeway was..."
